### DISCUSSION TITLE
Possible to use latest commit?

### DISCUSSION THREAD

**Discussion by tryconfident** [Flinesoft/BartyCrouch#271] [Q&A] [answered]
The most recent commit on main includes a fix I need to use DeepL translations; will there be a new release available for Brew soon, or is there a way I can brew install just that commit?

**Comment by Jeehut** [comment, marked as answer]
Homebrew works with "Formula" files. You could fork BartyCrouch and edit the tap formula file I provide in the repo [here](https://github.com/FlineDev/BartyCrouch/blob/main/Formula/bartycrouch.rb) and then tell Homebrew to install that formula directly on your local machine. I can't promise it's easy to switch to a custom formula and back once I had time to cut a release though, Homebrew doesn't officially support installing latest head, it always requires a release.

As I'm currently traveling throughout Japan, I can't make any promises, but I do have making a new BartyCrouch release in my TODO list already. Might take another week or so though. I want to also tackle the issue with Xcode 13 still not working properly.

**Reply by tryconfident** [reply]
Thank you! Safe travels. 😄
